test/mocks/providers/provider_post_page.dart

```diff
@@ -51,3 +53,23 @@ ProviderScope postPageProvider(
     child: postPage,
   );
 }
+
+ProviderScope customPostOverviewPage(PostOverview post) {
+  final customPostApp = MaterialApp(
+    onGenerateRoute: generateRoute,
+    home: PostPage(
+      postOverview: post,
+    ),
+  );
+
+  final app = ProviderScope(
```

Comment:
Here could we directly return the ProviderScope in order to stay consistent with the definition of "postPageProvider" above?

Reply:
I don't really like the way it was done above, so this is why I did it differently, but I didn't want to refactor it because it was not related to this PR...

Anyway, as it really is only a detail, I still applied your comment in aa0a976.